throw "there was a bad thing that happened!";

console.log("Hello");

// Any code after this error would never run.

/*
throw "there was a bad thing that happened!";
^
there was a bad thing that happened!
(Use `node --trace-uncaught ...` to show where the exception 
was thrown)
*/
